### ISSUE TITLE
Create user via LDAP Authentication throws 500 error

### ISSUE THREAD

**Issue by bandtank** [espocrm/espocrm#1561]
When I setup a new EspoCRM instance (5.7.11) using LDAP Authentication, I clicked the option to have Espo create the user based on the returned LDAP profile. It did eventually work, but not the first time. Please see the detailed information I posted in [this](https://forum.espocrm.com/forum/installation-and-upgrade-help/54712-ldap-auth-not-working-user-found-but-error-is-wrong-username-password) thread.

In short, the login flow threw a 500 after it tried to create the user, but the user was successfully created. Immediately clicking the login button again worked as expected. It looks like a null is being passed to a constructor instead of the newly created user object if the user does not exist before the login is attempted for the first time.

**Comment by yurikuzn** [comment]
https://github.com/espocrm/espocrm/commit/30afaa56ab873c32dd80dff92cd498eba82f54fc
